.github/workflows/ci-cd.yml

```diff
@@ -131,9 +139,40 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           target: runtime-base
-          push: true
-          registry_user: ${{ secrets.ECOSYSTEM_SERVICE_USER_DOCKER_HUB }}
-          registry_token: ${{ secrets.ECOSYSTEM_ACCESS_TOKEN_DOCKER_HUB }}
+          push: false
+          load: true
+      - name: Get EAP from base image
+        id: get_eap_file_name
+        run: |
+          docker cp $(docker create "${{ steps.meta.outputs.full_name }}"):/opt/app .build
+          export EAP_FILE=$(find .build -type f -name "*.eap" -printf "%f\n")
+          delimiter="$(openssl rand -hex 8)"
```

Comment:
I don't really follow what the delimiter does here, also you don't seem to use it after this step so is the output needed?